src/en/contributing/release_process.md

```diff
@@ -0,0 +1,20 @@
+Branching and Releases
+===
+
+## Branches
+
+In general, our handling of branches is the one described in [A stable mainline branching model for Git](https://www.bitsnbites.eu/a-stable-mainline-branching-model-for-git/). One difference is that we avoid rebase, and instead merge the base branch into the current working branch. This helps to avoid force pushes and conflicts.
+
+## Releases
+
+- For major release: make a new branch `release/v0.x`
+- For minor release: cherry-pick desired changes onto `release/v0.x` branch
+- Make a beta or release candidate version with `docker/prod/deploy.sh`
+- Do the same for `lemmy-ui`: `./deploy.sh 0.x.0-rc-x`
+- Deploy to federation test instances
+    -  keeping one instance at the last stable version to test federation compatibility (automate this with ansible)
+    -  eg: `ansible-playbook -i federation playbooks/site.yml --vault-password-file vault_pass -e "override_lemmy_image=dessalines/lemmy:0.x.0-rc.x override_lemmy_ui_image=dessalines/lemmy-ui:0.x.0-rc.x"`
```

Comment:
The override affects all instances, so it doesnt allow us to keep voyager on stable. Not sure what would be a good way to fix that.

Reply:
Ya not sure any way to automate that other than to make voyager always a manual deploy.